src/components_redux/Radios.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+
+export default class Radios extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  radioChange = event => {
+    this.setState({
+      type: event.target.value
+    });
+    this.props.changeType(event.target.value);
```

Comment:
Would be more logical if this prop would be called `onChange` like for normal components, like `input`.

Reply:
I named it like this because we discuss before I you told me to name it changeType.

Reply:
In inputFields it should be `changeType`, in Radios it should be `onChange`.
Radios should be a "general" component, where you do not care about what is changed. While inputFields is a specific component where it is changing that specific property with the name type.